docs/section22/Rule22-22.md

```diff
@@ -0,0 +1,50 @@
+
+# CHW&CW - Rule 22-22  
+
+**Rule ID:** 22-22  
+**Rule Description:** The baseline building design’s chiller plant shall be modeled with chillers having the type as indicated in Table G3.1.3.7 as a function of building peak cooling load.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.1 Type and Number of Chillers (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** Table G3.1.3.7  
+**Function Call:**  
+
+1. get_baseline_system_types()
+
+**Applicability Checks:**  
+
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to the next applicability check: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:  
+
+- For building peak cooling load in B_RMR: `building_peak_load_b = building_b.peak_cooling_load:` (Note XC, treat all buildings in one RMR as one building; pending schema element for building peak load.)
+
+  - Check if building peak cooling load is less than 600 tons: `if building_peak_load_b < 600 * CONVERSION(TON_TO_BTUH):`
+
```

Comment:
Need to double-check conversion, whether it should be Btu/hr to Ton or W to Ton instead.